sources/packages/web/src/views/student/StudentProfile.vue

```diff
@@ -97,19 +125,24 @@ export default {
           givenNames: studentAllInfo.value.firstName,
           dateOfBirth: studentAllInfo.value.birthDateFormatted,
           mode: FormModes.edit,
+          identityProvider: AuthService.shared.userToken?.IDP,
```

Comment:
why are we doing null check for the user token here?

Reply:
The `AuthService.shared.userToken` is defined as a nullable because right at the start of the application it will not be populated.
At this moment it will not be null but the `AuthService.shared.userToken!.IDP` is not recommended.